docs/metasploit-framework.wiki/Metasploit-Guide-MySQL.md

```diff
@@ -79,6 +79,157 @@ run cidr:/24:mysql://user:pass@192.168.222.0 threads=50
 run cidr:/24:mysql://user@192.168.222.0 threads=50 pass_file=./wordlist.txt
 ```
 
+### Obtaining an Interactive Session on the Target
+
+The CreateSession option in `scanner/mysql/msql_login` allows you to obtain an interactive session
+for the MySQL client you're connecting to. The run command with CreateSession
+set to true should give you an interactive session:
+
+```msf6
+run rhost=127.0.0.1 rport=4306 username=root password=password createsession=true
+
+[+] 127.0.0.1:4306        - 127.0.0.1:4306 - Found remote MySQL version 11.2.2
+[+] 127.0.0.1:4306        - 127.0.0.1:4306 - Success: 'root:password'
+[*] MySQL session 1 opened (127.0.0.1:53241 -> 127.0.0.1:4306) at 2024-03-12 12:40:46 -0500
+[*] 127.0.0.1:4306        - Scanned 1 of 1 hosts (100% complete)
+[*] Auxiliary module execution completed
+msf6 auxiliary(scanner/mysql/mysql_login) > sessions -i -1
+[*] Starting interaction with 1...
+
+mysql @ 127.0.0.1:4306 () >
```

Comment:
```suggestion
mysql @ 127.0.0.1:4306 >
```
should be able to remove that from here with this PR https://github.com/rapid7/metasploit-framework/pull/18979